src/components/Coaching/CoachingDetail/Activity/Activity.tsx

```diff
@@ -723,6 +474,57 @@ export const Activity: React.FC<ActivityProps> = ({
             </>
           )}
         </ActivitySection>
+        <ActivitySection data-testid="CurrentRealityPartnerFinancial">
+          {loading ? (
+            <MultilineSkeleton lines={1} width="90%" height="90%" />
+          ) : (
+            <StatsLargeNumber>
+              {data?.accountListAnalytics.contactsByStatus.financial}
+            </StatsLargeNumber>
+          )}
+          <Link
+            href={contactsLink({
+              status: [ContactFilterStatusEnum.PartnerFinancial],
+            })}
+            accountListType={accountListType}
+          >
+            <SectionTitle>{t('Financial Partners')}</SectionTitle>
+          </Link>
+        </ActivitySection>
+        <ActivitySection data-testid="CurrentRealityPartnerSpecial">
+          {loading ? (
+            <MultilineSkeleton lines={1} width="90%" height="90%" />
```

Comment:
These skeletons push the text down. Is there a way to make their height just the height of the number?